packages/asset-service/src/service/AssetService.ts

```diff
@@ -47,6 +47,11 @@ type ByTokenIdArgs = {
   tokenId?: string
 }
 
+type DescriptionData = {
+  description: string
+  isTrusted: boolean
+}
```

Comment:
I know this is super-flipping-paranoid, but maybe it'd be nice to only return frozen objects from `description()`. IDK what kind of exploit code might try setting `isTrusted = true` on an existing description but I'd sleep better at night if it wasn't even something to worry about.

```suggestion
type DescriptionData = Readonly<{
  description: string
  isTrusted: boolean
}>
```

Reply:
this is a really good point and should be included

Reply:
Done